php_02/php_02.php

```diff
@@ -0,0 +1,96 @@
+<?php
+	$errors = array();
+	$valid = array();
+	$errorCount = 0;
+	
+	if($_POST)
+	{
+		$title = $_POST['title'];
+		$titleLen = strlen($title);
+		if(!$title)
+		{
+			$errors['title'] = 'Името на курса е задължително!<br/>';
+			echo $errors['title'];
+			$errorCount++;
+		}
+		else if ($titleLen > 150)
+		{
+			$errors['title'] = 'Името на заглавието трябва до 150 символа!<br/>';
+			echo $errors['title'];
+			$errorCount++;
+		}
+		else
+		{
+			$valid ['title'] = $title;
+		}
+		
+		$teacher = $_POST ['teacher'];
+		if(!$teacher)
+		{
+			$errors['teacher'] = 'Името на преподавателят е задължително!<br/>';
+			echo $errors['teacher'];
+			$errorCount++;
+		}
+		else if (strlen($teacher) > 200)
+		{
+			$errors['teacher'] = 'Името на преподавателят трябва до 200 символа!<br/>';
+			echo $errors['teacher'];
+			$errorCount++;
+		}
+		else
+		{
+			$valid ['teacher'] = $teacher;
+		}
+		
+		$description = $_POST ['description'];
+		if(!$description)
+		{
+			$errors['description'] = 'Описанието е задължително!<br/>';
+			echo $errors['description'];
+			$errorCount++;
+		}
+		else if (strlen($description) > 200)
+		{
+			$errors['description'] = 'Описанието трябва да е поне 10 символа!<br/>';
+			echo $errors['description'];
+			$errorCount++;
+		}
+		else
+		{
+			$valid ['description'] = $description;
+		}
+		
+		$group = $_POST ['group'];
```

Comment:
What makes you think that the backend has received valid information? You should never trust the information that the frontend is sending you. Please add a validation. You may want to read more on my comment here: https://github.com/DianaGeorgievaa/WEB2020-php/pull/1#discussion_r392711585